create-topics.sh

```diff
@@ -27,15 +27,26 @@ if $start_timeout_exceeded; then
 fi
 
 # Expected format:
-#   name:partitions:replicas:cleanup.policy
-IFS=','; for topicToCreate in $KAFKA_CREATE_TOPICS; do
+#   name:partitions:replicas:cleanup.policy:<key>=<value>:<key>=<value>...
+IFS=';'; for topicToCreate in $KAFKA_CREATE_TOPICS; do
```

Comment:
Not needed since https://github.com/wurstmeister/kafka-docker/pull/305, so this can be made as a non-breaking change.